.cron.yml

```diff
@@ -19,7 +19,10 @@ jobs:
           type: decision-task
           treeherder-symbol: Nd-gp
           target-tasks-method: nightly-on-google-play
-      when: []  # Manual push only
+      when:
+          - {weekday: Monday, hour: 6, minute: 0}
```

Comment:
`weekday` is supported per https://searchfox.org/mozilla-central/rev/97cb0a90bd053de87cd1ab7646d5565809166bb1/taskcluster/taskgraph/cron/schema.py#79